src/box/replication.cc

```diff
@@ -116,6 +116,7 @@ replication_init(int num_threads)
 	rlist_create(&replicaset.applier.on_wal_write);
 
 	rlist_create(&replicaset.on_ack);
+	rlist_create(&replicaset.on_relay_thread_start);
```

Comment:
Also need to destroy the trigger list in `replication_free()`.

Reply:
Fixed.